src/styles/global.css

```
@@ -2530,6 +2530,49 @@ body.blog-page {
+}
+
+.project-copy .unit-exhibit > div + div {
+  border-top: 1px dotted var(--rule);
+}
+
+.project-copy .unit-exhibit dt {
+  margin: 0;
+  font-size: 0.75rem;
+  letter-spacing: 0.08em;
+  text-transform: uppercase;
+  color: color-mix(in srgb, currentColor 62%, transparent);
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)